lib/bowling.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+
+class Bowling
+  attr_reader :total_score
+
+  def initialize(input: $stdin, output: $stdout)
+    @frames = []
+    @total_score = 0
+    @input = input
+    @output = output
+  end
+
+  def play
```

Comment:
Good approach to running the scorecard, succinct code and easy to use in the terminal, and gives a clear printout each time.